Cards/MadGraph5_aMCatNLO/QCD/QCD0B-4Jets-BPSFilter_HT-1000to1200_madgraphMLM-pythia8/QCD0B-4Jets-BPSFilter_HT-1000to1200_madgraphMLM-pythia8.json

```diff
@@ -0,0 +1,20 @@
+{
+    "gridpack_submit": false,
+    "gridpack_path" : "QCD/QCD-4Jets_HT-1000to1200_madgraphMLM-pythia8",
```

Comment:
How do we group the gridpacks? I think for Powheg I used Powheg/TT/gridpack.tgz.

Reply:
what do you mean by group?

for powheg it should work with
"TT/TTto2L2Nu_blahblalh"

Reply:
this automatically looks for madgraph directory gridpacks since this card is inside madgraph dir

Reply:
Let me re-phrase: Is this a path or does it include the gridpack name ? 
I used SHORTPROCESS/OURNAME.tgz
Do you store gridpacks now :
SHORTPROCESS/OURNAME/OURNAME.tar.gz ?